resources/views/themes/default1/layouts/front/master.blade.php

```diff
@@ -467,12 +467,23 @@
             @if(Session::has('success'))
 
             <div class="container">
+                
 
                   {!!Session::get('success')!!}
             </div>
 
             @endif
+            
+               @if(Session::has('Success'))
```

Comment:
why was this added? we already have something to display success messages also a gurmeen PR is raised

Reply:
For this @Gurmeenkour99 raised the PR, once that is merged I will rebase it.